server/config.coffee

```diff
@@ -33,7 +33,8 @@ config =
             americano.errorHandler
                 dumpExceptions: true
                 showStack: true
-            americano.static path.join __dirname, '/../client/public'
+            americano.static path.join __dirname, '../static'
```

Comment:
Why `client/public` was moved to `static`? I think it was logic to have client-side assets in the `client` folder.